spec.html

```diff
@@ -4259,7 +4259,7 @@
         1. Let _keys_ be ? _O_.[[OwnPropertyKeys]]().
         1. If _level_ is `"sealed"`, then
           1. Repeat for each element _k_ of _keys_,
-            1. Perform ? DefinePropertyOrThrow(_O_, _k_, PropertyDescriptor{ [[Configurable]]: *false*}).
+            1. Perform ? DefinePropertyOrThrow(_O_, _k_, PropertyDescriptor{[[Configurable]]: *false*}).
```

Comment:
Maybe white space was added to attribute sets (94 of the 108 matches) because the sets almost always appear in continuous text as opposed to algorithms? Anyway, I'm not opposed to add white space around the curly braces of record objects, but for now I'd prefer if we keep the PR as is.